htdocs/core/modules/expensereport/doc/pdf_standard.modules.php

```diff
@@ -358,7 +358,7 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
                 while ($i < $nblines) {
 					$pdf->SetFont('', '', $default_font_size - 2);   // Into loop to work with multipage
 					$pdf->SetTextColor(0, 0, 0);
-                    if (empty($showpricebeforepagebreak)) {
+                    if (empty($showpricebeforepagebreak) && $i !== $nblines - 1) {
```

Comment:
$i !== $nblines - 1    is difficult for me to interpret. Can you add parenthesis ? Is it  $i !== ($nblines - 1)  ?  Because binary comparison seems to be done in priority.

Reply:
I checked the manual: `-` has precedence over `!==`, which has precedence over `&&`, so
```php
empty($showpricebeforepagebreak) && $i !== $nblines - 1
```
would be the same as
```php
empty($showpricebeforepagebreak) && ($i !== ($nblines - 1))
```
But I agree that the latter is much more readable, so I changed it as requested.